examples/Basic usage - Functional.ipynb

```diff
@@ -128,7 +128,9 @@
     "\n",
     "The third dimension contains all the decontaminated signals from the ROI and its surrounding neuropil.\n",
     "The 0-th entry of this is the signal which most closely corresponds to the raw signal within the ROI.\n",
-    "The other signals are the (isolated) contaminants."
+    "The other signals are the (isolated) contaminants.\n",
+    "\n",
+    "We now first plot all the ROI signals for a given trial in a single plot:"
```

Comment:
I don't like that the sentence starts with "now first", which are two (redundant? conflicting? unsure) ways of describing the current time. Pick one!

Also, I don't like that the sentence ends in a colon.